cvat-ui/src/utils/opencv-wrapper/opencv-wrapper.ts

```diff
@@ -89,6 +90,14 @@ export class OpenCVWrapper {
             intelligentScissorsFactory: () => new IntelligentScissorsImplementation(this.cv),
         };
     }
+
+    public get hist(): HistogramEqualization {
+        if (!this.initialized) {
+            throw new Error('Need to initialize OpenCV first');
+        }
+
+        return new HistogramEqualizationImplementation(this.cv);
+    }
```

Comment:
Conceptually, I see it followings:

```js
public get imgproc() {
    if (!this.initialized) {
            throw new Error('Need to initialize OpenCV first');
     }

     return {
         hist: return new HistogramEqualizationImplementation(this.cv),
     }
}
```

What do you think? imgproc is a module of OpenCV where histogram equalization located

Reply:
Done